NuGet.Docs/Create/uwp-create.md

```diff
@@ -18,10 +18,19 @@ In this case you need to add the uap10.0 TxM to your package. Create a new direc
 
 In this case you would add the new dotnet TxM to your package. Unlike other TxMs dotnet doesn’t imply a surface area or platform. It is stating that your package will work on any platform that your dependencies work on. When building a package with the dotnet TxM you are likely to have many more TxM specific dependencies in your NuSpec, as you will need to define the BCL packages you depend on, such System.Text, System.Xml, etc. The locations that those dependencies work on define where your package will work.  
 
+### How do I find out my dependencies?
+
+There are two ways to figure out which dependencies to list:
+
+1. Use ILDasm to look at your dll to see what assemblies are actually needed at runtime. Then determine which NuGet package that comes from. This is the hard way.
+2. Use the [NuSpec Dependency Generator](https://github.com/onovotny/ReferenceGenerator) tool created by one of the Windows Platform Developer MVPs. The tool automates the process and updates your .nuspec file with the depependant packages on build. It is available via a NuGet package, [NuSpec.ReferenceGenerator](https://www.nuget.org/packages/NuSpec.ReferenceGenerator/).
```

Comment:
Can you please remove the created by section, and replace with

Use the [NuSpec Dependency Generator](https://github.com/onovotny/ReferenceGenerator) **3rd party**  tool